packages/ui/src/theme/css/component/tabs.scss

```diff
@@ -57,7 +58,7 @@
   &:focus-visible {
     outline: var(--amplify-components-tabs-item-border-width)
       var(--amplify-components-tabs-item-border-style)
-      var(--amplify-components-tabs-item-border-color);
+      var(--amplify-components-tabs-item-focus-visible-outline-color);
```

Comment:
The existing border-color we were using for the focus-visible outline for Tab buttons did not meet 3:1 contrast so it's been updated to use a new token that references our existing `colors.border.focus` color that we use elsewhere for focus ring color.